daemon/api_test.go

```diff
@@ -122,11 +123,36 @@ func (s *apiSuite) TearDownTest(c *check.C) {
 	s.stateOverlord.Stop()
 }
 
+func (s *apiSuite) mkStoreYaml(c *check.C, pkgType snap.Type) {
+	// creating the part to get its manifest path is cheating, a little
+	info := &snap.Info{
+		Name:      "foo",
+		Developer: "bar",
+		Version:   "1",
+		Type:      pkgType,
+	}
+
+	content := `
+name: foo
+origin: bar
+version: 1
+revision: 999999999999
+type: app
+description: |-
+  bla bla bla
+iconurl: http://i.stack.imgur.com/i8q1U.jpg
+downloadsize: 5554242
+`
+	p := snappy.RemoteManifestPath(info)
+	c.Assert(os.MkdirAll(filepath.Dir(p), 0755), check.IsNil)
+	c.Assert(ioutil.WriteFile(p, []byte(content), 0644), check.IsNil)
+
+}
+
 func (s *apiSuite) mkInstalled(c *check.C, name, developer, version string, active bool, extraYaml string) {
```

Comment:
Just pointing out that developer is still part of the signature of this test helper method